test/libsolidity/semanticTests/operators/custom/function_bound_to_operator_as_global_and_non_global.sol

```diff
@@ -0,0 +1,30 @@
+==== Source: s1.sol ====
+type Int is int;
+
+using {add as +} for Int global;
+using {add as +} for Int;
```

Comment:
Maybe we would be better off reporting an error here? I could see a use case if you could redefine it in a different file. But if you can only attach operators in the file where the type is defined, it's either a mistake or done intentionally to deceive. E.g. someone could put the non-global version at the top of a long file and sneak through the global version somewhere down between functions.

Reply:
Oh, by the way, do we already have a test that shows that you cannot bind an operator to a type defined in a separate file?

Reply:
Just to clarify, I can only attach operators in the file where the type is defined when I do this with `global`. There is no such restriction for non-global attaching. Moreover, in this particular scenario, everything works fine only because we bind the same function twice. If you try to bind two different functions, compilation ends up with errors on operator resolving and it makes no difference at this point if the operators are defined globally or not.

Reply:
Ah, right! It's ok then.